addons/hr_holidays/models/hr.py

```diff
@@ -190,13 +190,14 @@ def _search_absent_employee(self, operator, value):
 
     def write(self, values):
         res = super(Employee, self).write(values)
+        today_date = datetime.datetime.utcnow().date()
```

Comment:
Usually we use ``fields.Datetime.now()``.

Everything should be correctly managed into it.